templates/Web/_composition/Node/Project.Node/.template.config/template.json

```
@@ -1,43 +1,41 @@
-﻿{
-  "$schema": "http://json.schemastore.org/template",
-  "author": "Microsoft",
-  "classifications": ["Universal"],
-  "name": "NodeDefault",
-  "shortName": "NodeDefault",
-  "groupIdentity": "wts.Proj.NodeDefault",
-  "identity": "wts.Proj.NodeDefault",
-  "description": "",
-  "tags": {
-    "language": "Any",
-    "type": "project",
-    "wts.type": "project",
-    "wts.projecttype": "FullStackWebApp",
-    "wts.frontendframework": "all",
-    "wts.backendframework": "Node",
-    "wts.platform": "Web",
-    "wts.version": "1.0.0",
-    "wts.compositionOrder": "2",
-    "wts.licenses": "[Node](https://github.com/nodejs/node/blob/master/LICENSE)|[Express](https://github.com/expressjs/express/blob/master/LICENSE)|[Express Generator](https://github.com/expressjs/generator/blob/master/LICENSE)"
-  },
-  "sourceName": "wts.NodeDefault",
-  "preferNameDirectory": true,
-  "PrimaryOutputs": [],
-  "symbols": {
-    "wts.projectName": {
-      "type": "parameter",
-      "dataType": "string",
-      "replaces": "Param_ProjectName"
-    },
-    "wts.ProjectNameToLower":
-    {
-      "type": "generated",
-      "generator": "casing",
-      "parameters": {
-        "source":"wts.projectName",
-        "toLower": true
-      },
-      "replaces": "Param_ProjectNameToLower"
-    }
-  },
-  "postActions": []
-}
+﻿{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": ["Universal"],
+  "name": "Project.Node",
+  "shortName": "Project.Node",
+  "groupIdentity": "wts.Project.Default",
+  "identity": "wts.Project.Node",
+  "description": "",
+  "tags": {
+    "language": "Any",
+    "type": "item",
+    "wts.type": "composition",
+    "wts.platform": "Web",
+    "wts.version": "1.0.0",
+    "wts.compositionOrder": "1",

```

Just in case it helps. Backend template

```suggestion
    "wts.compositionOrder": "2",

```